application/controllers/admin/ParticipantsAction.php

```diff
@@ -1463,7 +1463,15 @@ public function changeAttributeEncrypted()
                         $aUpdateData['value'] = LSActiveRecord::encryptSingle($attribute->value);
                     }
                     if (!empty($aUpdateData) && $aUpdateData['value'] !== null) {
-                        $oDB->createCommand()->update('{{participant_attribute}}', $aUpdateData, "attribute_id='" . $attributeId . "' AND participant_id = '" . $attribute->participant_id . "'");
+                        $oDB->createCommand()->update(
+                            '{{participant_attribute}}',
+                            $aUpdateData,
+                            'attribute_id=:attribute_id AND participant_id=:participant_id',
+                            array(
+                                'attribute_id' => $attributeId,
```

Comment:
Not sure if correct. Should be
```php
':attribute_id' => $attributeId,
```
?

Reply:
It's about colon or not in the array key `:attribute_id` vs `attribute_id` :)

Reply:
> It's about colon or not in the array key `:attribute_id` vs `attribute_id` :)

Yes : i think you're right ! Totally right !

Seems not tested a lot …

Reply:
Well spotted @olleharstedt. Fix was pushed to the sonarcubefixes branch.